server/src/main/java/com/paypal/selion/grid/IOSDriverJarSpawner.java

```diff
@@ -45,12 +45,12 @@ public IOSDriverJarSpawner(String[] args, ProcessLauncherOptions options) {
         setType(InstanceType.IOS_DRIVER);
     }
 
-    public static final void main(String[] args) {
+    public static void main(String[] args) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Consider using varargs for methods or constructors which take an array the last parameter.](https://www.codacy.com/app/SeLion/paypal-SeLion/file/2103142562/issues/source?bid=2928425&fileBranchId=3110989#l48)